include/miopen/config.h.in

```diff
@@ -78,11 +78,65 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
+#cmakedefine01 MIOPEN_OFFLINE_COMPILER_PATHS_V2
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+// Include rocm-core header for get ROCm Install Path Method
+#include <rocm-core/rocm_getpath.h>
+
+// Include header for PATH_MAX
+#if defined(__linux__)
+#include <linux/limits.h>
+#endif
+#if defined(_WIN32)
+#include <windows.h>
+#ifndef PATH_MAX
+#define PATH_MAX MAX_PATH
+#endif
+#endif
+
+// Max Buffer Len to hold the Path
+#define PATH_MAX_LEN (PATH_MAX + 1)
+
+// Flags to help identify/differentiate Definitions for each MIOPEN Compiler Paths
+typedef enum
+{
+    MIOPEN_AMDGCN_Flag          = 0,
+    MIOPEN_OC_COMPILER_Flag     = 1,
+    MIOPEN_HIP_COMPILER_Flag    = 2,
+    MIOPEN_OFFLOAD_BUNDLER_Flag = 3,
+    MIOPEN_CMP_MAX_Flag         = 4 // Should be last Flag definition
+} MIOPEN_CMP_Flags_t;
```

Comment:
Non of the MIOpen C API requires this enumeration type. We should move it out of this file.

Reply:
+1 and this is covered at https://github.com/ROCm/MIOpen/pull/2694#discussion_r1486814062

Reply:
Thanks. Updated in latest version